src/lib/is-browser.ts

```diff
@@ -1,5 +1,17 @@
-const isStandardBrowserEnv = () =>
-	typeof window !== 'undefined' && typeof window.document !== 'undefined'
+const isStandardBrowserEnv = () => {
+	if (typeof window !== 'undefined') {
+		if (
+			navigator.userAgent.toLowerCase().indexOf(' electron/') > -1 ||
+			Object.prototype.hasOwnProperty.call(process.versions, 'electron')
+		) {
+			return false
+		}
```

Comment:
@axi92 Maybe it was correct to keep the `&&` now that I think about this again, because seems this happens only when there is no context isolation: in that case the isBrowser returns true because window is defined but there is also nodejs available.

 I think reverting back to `&&` is the correct way. Please add a comment that explains this above the code you added to make it clear to everyone looking at this.